lib/packer/builders/amazon.rb

```diff
@@ -140,6 +140,30 @@ def vpc_id(id)
         self.__add_string('vpc_id', id)
       end
 
+      def communicator(comm)
```

Comment:
Also, I should probably add in the ssh-equivalents as well, eh? https://packer.io/docs/templates/communicator.html